setup/src/Magento/Setup/Model/ConfigOptionsList/DriverOptions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Magento\Setup\Model\ConfigOptionsList;
+
+use Magento\Framework\Config\ConfigOptionsListConstants;
+
+class DriverOptions
+{
+    /**
+     * @params array $options
+     * @return array
+     */
+    public function getDriverOptions($options)
+    {
+        $driverOptionKeys = [
+            ConfigOptionsListConstants::KEY_MYSQL_SSL_KEY => ConfigOptionsListConstants::INPUT_KEY_DB_SSL_KEY,
+            ConfigOptionsListConstants::KEY_MYSQL_SSL_CERT => ConfigOptionsListConstants::INPUT_KEY_DB_SSL_CERT,
+            ConfigOptionsListConstants::KEY_MYSQL_SSL_CA => ConfigOptionsListConstants::INPUT_KEY_DB_SSL_CA,
+            ConfigOptionsListConstants::KEY_MYSQL_SSL_VERIFY => ConfigOptionsListConstants::INPUT_KEY_DB_SSL_VERIFY
+        ];
+        $driverOptions = [];
+        foreach ($driverOptionKeys as $configKey => $driverOptionKey) {
+            if ($this->optionExists($options, $driverOptionKey)) {
+                $driverOptions[$configKey] = $options[$driverOptionKey];
+            }
+        }
+        return $driverOptions;
+    }
+
+    /**
+     * @param array $options
+     * @param string $driverOptionKey
+     * @return bool
+     */
+    protected function optionExists($options, $driverOptionKey)
```

Comment:
Due to [Magento backward-compatible development guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't declare protected methods. Please make this method private.